tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -966,6 +966,21 @@ handle_build() {
 	fi
 	tar tvf "${final_tarball_path}"
 
+	case ${build_target} in
+		kernel*-confidential|kernel-sev)
+			local modules_final_tarball_path="${workdir}/kata-static-${build_target}-modules.tar.xz"
+			if [ ! -f "${modules_final_tarball_path}" ]; then
+				local modules_dir=$(get_kernel_modules_dir ${kernel_version} ${kernel_kata_config_version})
```

Comment:
A small suggestion is to squash this commit with the first one that you are introducing the function.